### ISSUE TITLE
[member-naming] Should match constructor args

### ISSUE THREAD

**Issue by antoine-pous** [typescript-eslint/typescript-eslint#765]
**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/no-parameter-properties": ["error", {
      "allows": [
        "private"
      ]
    }],
    "@typescript-eslint/member-naming": ["error", {
      "private": "^__"
    }]
  }
}
```

```TS
class Test {
  constructor(private myArg: string) {
  
  }
}
```

**Expected Result**
It should emit an error, due to the configuration `myArg` must be `__myArg`

**Actual Result**
No error

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `1.13.0` |
| `@typescript-eslint/parser`        | `1.13.0` |
| `TypeScript`                       | `3.4.5` |
| `ESLint`                           | `6.1.0` |
| `node`                             | `12.6.0` |
| `npm`                              | `6.10.2` |
